### DISCUSSION TITLE
image recognition from a folder

### DISCUSSION THREAD

**Discussion by Ryoperso** [exadel-inc/CompreFace#1004] [Q&A]
Hello,
I have just discovered your solution and I think it is fantastic.
The installation went very well and the interface is very intuitive.

On the other hand, I think there is a part that I did not understand or that I execute badly. I expose you my problem:

OBJECTIVES:
1- I want to be able to do a facial analysis of a file containing hundreds of photos of faces in order to highlight the images of faces that are identical
2- and then I want to be able to compare these hundreds of reference images with another target image to check if the latter is present

PROBLEMATIC :
Except that in compreface, I can't integrate a folder. I have the impression that I have to integrate the pictures image by image. Which is quite time consuming.

Do you have a solution please?
thank you
Sincerely,

**Comment by pospielov** [comment]
Hello,
If I understand correctly, you are trying to use CompreFace UI to solve your problem.
If yes, I don't think there is such a possibility in CompreFace UI. You can upload several images at once in the face collection, but the test page is intended to test only one image at once. 
It's not very hard to create a script that sends all images from the folder to CompreFace REST API, but it will require at least basic programming skills.
